ocs_ci/ocs/ui/base_ui.py

```
@@ -1638,42 +1646,42 @@ def navigate_to_local_cluster(**kwargs):
-        local_cluster_item.click()
-    except TimeoutException:
-        current_url = SeleniumDriver().current_url
-        logger.warning(
-            f"'All Clusters' dropdown not found within {timeout}s "
-            f"(current URL: {current_url}). "
-            "Checking whether local-cluster page is already loaded."
-        )
-        wait_for_element_to_be_visible(acm_page_loc["local-cluster_dropdown"], timeout)
+    logger.info(
+        f"Perspective '{current_perspective}' is not the local cluster. "
+        "Opening the perspective switcher to select the local cluster."
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)